Test/Resolve-WhiskeyPowerShellModule.Tests.ps1

```diff
@@ -56,6 +56,16 @@ function GivenModuleDoesNotExist
     Mock -CommandName 'Find-Module' -ModuleName 'Whiskey'
 }
 
+function GivenModuleInstalledGlobally
+{
+    Mock -CommandName 'Test-WhiskeyPowershellModule' -ModuleName 'Whiskey' -MockWith { $true }
+}
+
+function GivenModuleNotInstalledGlobally

```

Comment:
Rename these functions to `GivenPkgMgmtModulesInstalledGlobally` to be clear about what modules we're expecting to be available globally.